<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: none;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="drop_shadow_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="drop_shadow_frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(42, 44, 111, 255), stop:0.521368 rgba(28, 29, 73, 255));
	border-radius: 12px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QFrame" name="title_bar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_boutton">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>15</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Noto Serif</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 255, 255)</string>
               </property>
               <property name="text">
                <string>Modern Computer Monitoring</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_title">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="maximumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius:8px;
	background-color: rgb(85,255,127);
}

QPushButton:hover {
	background-color: rgba(85,255,127,156);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="maximumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius:8px;
	background-color: rgb(255,170,0);
}

QPushButton:hover {
	background-color: rgba(255,170,0,156);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="maximumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius:8px;
	background-color: rgb(255,0,0);
}

QPushButton:hover {
	background-color: rgba(255,0,0,156);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="content">
         <property name="styleSheet">
          <string notr="true">background-color: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background-color:none;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_content">
               <property name="styleSheet">
                <string notr="true">background-color:none;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QFrame" name="frame_2">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QFrame {
	border: 5px solid rgb(60,231,195);
	border-radius: 75px;
}

QFrame:hover {
	border: 5px solid rgb(60,180,195);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <property name="leftMargin">
                    <number>10</number>
                   </property>
                   <property name="topMargin">
                    <number>30</number>
                   </property>
                   <property name="rightMargin">
                    <number>10</number>
                   </property>
                   <property name="bottomMargin">
                    <number>30</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="styleSheet">
                      <string notr="true">
color: rgb(60,231,195);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>CPU USAGE</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <pointsize>22</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(220,220,200);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>25%</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="styleSheet">
                      <string notr="true">color: rgb(128,102,168);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>Intel I7 6700k </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_3">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QFrame {
	border: 5px solid rgb(60,231,195);
	border-radius: 75px
}

QFrame:hover {
	border: 5px solid rgb(60,180,195);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <property name="leftMargin">
                    <number>10</number>
                   </property>
                   <property name="topMargin">
                    <number>15</number>
                   </property>
                   <property name="rightMargin">
                    <number>10</number>
                   </property>
                   <property name="bottomMargin">
                    <number>15</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="styleSheet">
                      <string notr="true">
color: rgb(60,231,195);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>GPU USAGE</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="font">
                      <font>
                       <pointsize>22</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(220,220,200);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>90%</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="styleSheet">
                      <string notr="true">color: rgb(128,102,168);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>NVIDIA RTX 3070</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_12">
                     <property name="font">
                      <font>
                       <pointsize>18</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(220,220,200);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>50°C</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_4">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QFrame {
	border: 5px solid rgb(60,231,195);
	border-radius: 75px
}

QFrame:hover {
	border: 5px solid rgb(60,180,195);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <property name="leftMargin">
                    <number>10</number>
                   </property>
                   <property name="topMargin">
                    <number>30</number>
                   </property>
                   <property name="rightMargin">
                    <number>10</number>
                   </property>
                   <property name="bottomMargin">
                    <number>30</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_9">
                     <property name="styleSheet">
                      <string notr="true">
color: rgb(60,231,195);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>RAM USAGE</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_10">
                     <property name="font">
                      <font>
                       <pointsize>22</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(220,220,200);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>25%</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_11">
                     <property name="styleSheet">
                      <string notr="true">color: rgb(128,102,168);
border:none;</string>
                     </property>
                     <property name="text">
                      <string>Intel I7 6700k </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="credit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Serif</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 255, 255)</string>
            </property>
            <property name="text">
             <string>Réalisé par PEREIRA Jonathan</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
